examples/mnist/README.md

```diff
@@ -0,0 +1,8 @@
+Sample MNIST Graph to export as Tensorflow Hub Module
+------------------------------------------------------
+### To Export
+```bash
+$ python3 export.py
+```
+### to run The Tests
+- `$ bazel test E1_TFHub_Sample_Deploy:export_test`
```

Comment:
Please update this.

Reply:
Can you rephrase? didn't really get what you said.

Reply:
E1_TFHub_Sample_Deploy has no meaning here.

Reply:
Yep. Fixed :)